api/server.go

```diff
@@ -763,6 +764,19 @@ func (s *server) hostsHandler(jc jape.Context) {
 	jc.Encode(hosts)
 }
 
+func (s *server) proofIndexHandler(jc jape.Context) {
```

Comment:
Do we need this? Don't we have an endpoint that already lets you fetch a file contract including its state element? It would be a bit more usable to have an endpoint that you can fetch a contract from with its up-to-date proof instead of having to call another endpoint.

Reply:
I've been leaving them as nil because retrieving them requires multiple queries.  I guess at least for these endpoints:
```
		"GET    /outputs/siacoin/:id": srv.outputsSiacoinHandler,
		"GET    /outputs/siafund/:id": srv.outputsSiafundHandler,
		"GET    /contracts/:id":           srv.contractsIDHandler,
		"GET    /v2/contracts/:id":           srv.v2ContractsIDHandler,
```
it makes sense to set them.